store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -79,6 +81,52 @@ private String checkUnbalancedTags(String html) {
         }
         return html;
     }
+    
+    /** 
+     * A method to check & remove any any child parent comparator inside <style> tag 
+     * 
+     * @return the Formatted Html after removing child comparators(>)
+     */
+    private String removeChildComaparator(String html) {
+        if (StringUtil.isNullOrEmpty(html)) {
+            return html;
+        }
+        ArrayList<String> cssTagsList = new ArrayList<String>();
+        // in cssTagsList array we can add multiple tags 
+        cssTagsList.add("tbody");
+        cssTagsList.add("tr");
+        cssTagsList.add("div");
```

Comment:
right now we have tickets related to this tag... if we will get issue on other tag in future we just need to add in this cssTagsList array

Reply:
I see...  What to you think about adding a localconfig which specifies comma-separated tags like `tbody,tr,div`? Then we don't need to modify the java file just for adding a necessary tag. It might be useful for an administrator if it is configurable.

Reply:
If it causes another security concern, please ignore it.

Reply:
added uppercase & firstletter uppercase for all tags ... ( style,tbody,tr,div)

Reply:
it should be implemented using regex or something.
all patterns are valid as html: `TBODY, TBODy, TBOdy, TBody, Tbody, tbody`